hack/config.sh

```diff
@@ -0,0 +1,26 @@
+usage() {
```

Comment:
hashbang

Reply:
this is not the issue you are looking for....